src/test/java/org/sagebionetworks/bridge/udd/worker/BridgeUddWorkerTest.java

```diff
@@ -82,7 +86,7 @@ public void test() throws Exception {
         // mock Synapse packager
         SynapsePackager mockPackager = mock(SynapsePackager.class);
         when(mockPackager.packageSynapseData(same(mockSynapseToSchema), eq("test-health-code"),
-                any(BridgeUddRequest.class))).thenAnswer(invocation -> {
+                any(BridgeUddRequest.class), same(mockSurveyTableIdSet))).thenAnswer(invocation -> {
             // Second request is in March, that has no data. Third request is in August, that has data.
             BridgeUddRequest request = invocation.getArgumentAt(2, BridgeUddRequest.class);
             int startMonth = request.getStartDate().getMonthOfYear();
```

Comment:
In the immediate-term, we have no way of doing this, since the only output of B-UDD is an email, and we don't currently have a way to hook into that. In the long-term, this is tracked by https://sagebionetworks.jira.com/browse/BRIDGE-832